### ISSUE TITLE
audit-avt1-data-viz-chart-anatomy: role needs label

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1817]
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Data visualization - Chart anatomy](https://www.carbondesignsystem.com/data-visualization/chart-anatomy)
> This bug affects over 1000 total issues

## Detailed Description: 

The element with "complementary" role does not have a label

<img width="1680" alt="Screen Shot 2020-09-22 at 4 11 35 PM" src="https://user-images.githubusercontent.com/32720851/93938282-a2aec480-fcee-11ea-8bdd-a0b042b4e1c5.png">

## Why is this important? 
An element with a "complementary" role must have a label specified with an accessible name. This information helps people using assistive technologies to find and navigate the content.

## What to do: 
If there is visible text that appropriately describes the purpose of the content, set the element's aria-labelledby attribute to the id value of that text (example below);
